jmx-metrics/src/test/java/io/opentelemetry/contrib/jmxmetrics/MBeanHelperTest.java

```diff
@@ -116,6 +119,77 @@ void multiple() throws Exception {
             IntStream.range(0, 100).mapToObj(unused -> null).collect(Collectors.toList()));
   }
 
+  @Test
+  void transform() throws Exception {
+    String thingName = "io.opentelemetry.contrib.jmxmetrics:type=transform";
+    Thing thing = new Thing("someValue");
+    mbeanServer.registerMBean(thing, new ObjectName(thingName));
+    Map<String, Closure<?>> map =
+        Stream.of(
+                new Object[][] {
+                  {
+                    "SomeAttribute",
+                    Eval.me(
+                        "{mbean -> mbean.getProperty(\"SomeAttribute\") == 'someValue' ? 'otherValue' : 'someValue'}")
+                  },
+                })
+            .collect(Collectors.toMap(data -> (String) data[0], data -> (Closure<?>) data[1]));
+    MBeanHelper mBeanHelper = new MBeanHelper(jmxClient, thingName + ",*", true, map);
+    mBeanHelper.fetch();
+
+    assertThat(mBeanHelper.getAttribute("SomeAttribute"))
+        .hasSameElementsAs(Stream.of(new String[] {"otherValue"}).collect(Collectors.toList()));
+  }
+
+  @Test
+  void transformMultipleAttributes() throws Exception {
+    String thingName = "io.opentelemetry.contrib.jmxmetrics:type=transformMultiple";
+    Thing thing1 = new Thing("someValue", "anotherValue");
+    ObjectName mbeanName = new ObjectName(thingName);
+    mbeanServer.registerMBean(thing1, mbeanName);
+    Map<String, Closure<?>> map =
+        Stream.of(
+                new Object[][] {
+                  {
+                    "SomeAttribute",
+                    Eval.me(
+                        "{mbean -> mbean.getProperty(\"SomeAttribute\") == 'someValue' ? 'newValue' : 'someValue'}")
+                  },
+                  {
+                    "AnotherAttribute",
```

Comment:
Do you mind updating this to try to get a nonexistent property for error handling vetting? I think it repeats existing added coverage.

Reply:
In its current state I think that'll throw an AttributeNotFound exception, do we want to handle that error in the same manner as if a nonexistent property was passed to an instrument helper? I just followed the pattern of labelFuncs and the other closure params where it doesn't look like theres any additional error handling

Reply:
I think the same error behavior makes sense and it could be helpful to add coverage to better enforce that moving forward.